app.py

```diff
@@ -1,10 +1,146 @@
-from flask import Flask
+from flask import Flask, jsonify, request
+import traceback
+import random
+import datetime
+from test_mongo_data import Mdb
+from config import ITEM, CONTAINER, ARRANGEMENT, SNAPSHOT
+
 app = Flask(__name__)
+mdb = Mdb()
+
+def uniqueid():
+    seed = random.getrandbits(32)
+    while True:
+       yield seed
+       seed += 1
 
 
-@app.route('/')
-def hello():
+@app.route("/")
+def home_page():
     return "Arrange That!"
 
+@app.route('/arrangement', methods=['POST'])
+@app.route('/api/v1/arrangement', methods=['POST'])
+def add_arrangement():
+    content = request.json
+    try:
+        data = {}
+        name = content['name']
+
+        is_deleted = content['is_deleted']
+        items = add_item(content)
+        containers = add_container(content)
+        snapshots = add_snapshots(content)
+
+        data["_id"] = "a_"+str(next(uniqueid()))
+        data["name"] = name
+        data["item"] = items
+        data["snapshots"] = snapshots
+        data["is_deleted"] = is_deleted
+        data["containers"] = containers
+        data["timestamp"] = datetime.datetime.today().strftime("%a %b %d %X  %Y ")
```

Comment:
timestamp is going to be in seconds from the epoch. We can just have this done on the front end...